src/components/Notifications/BehaviorGroup/Wizard/SelectableEventTypeTable.tsx

```diff
@@ -74,33 +76,76 @@ const SelectableEventTypeTableSkeleton: React.FunctionComponent = () => {
   );
 };
 
+const SelectableEventTypeTableRow: React.FunctionComponent<{
+  event: SelectableEventTypeRow;
+  rowIndex: number;
+  onSelect?: (isSelected: boolean, event: EventType) => void;
+  selectionLoading?: boolean;
+}> = ({ event, rowIndex, onSelect, selectionLoading }) => {
+  const [isExpanded, setIsExpanded] = React.useState(false);
+
+  return (
+    <>
+      <Tr key={event.id}>
+        {event.description ? (
+          <Td
+            expand={{
+              rowIndex: rowIndex,
+              isExpanded,
+              onToggle: () => setIsExpanded(!isExpanded),
+              expandId: `expandable-toggle-${event.id}`,
+            }}
+          />
+        ) : (
+          <Td className="pf-u-display-none pf-u-display-block-on-md" />
```

Comment:
```suggestion
          <Td />
```
These class names do not work since they're missing `-v5` section. We can probably get rid of them and the whole condition can be simplified